contrib/debian/rules

```diff
@@ -7,6 +7,7 @@ export PYBUILD_NAME=subscription-manager
 export PYBUILD_DISABLE=test
 
 include /usr/share/dpkg/default.mk
+include /usr/share/dpkg/pkg-info.mk
```

Comment:
`default.mk` is included in the line before this, and already pulls all the other dpkg makefile scripts, including `pkg-info.mk` -- so the manual include is not needed (redundant)